spec/features/questions/show_spec.rb

```diff
@@ -0,0 +1,15 @@
+require "rails_helper"
+require 'pry'
+
+RSpec.describe "Show a question" do
+  context "Question exists" do
+    it "shows the question" do
+      question = RatingQuestion.create!(title: "hola")
```

Comment:
Move the creation of this question into a `let!` block inside the context. Setup like this should be done _before_ the test itself, in case there are multiple tests that share common setup.